pootle/static/js/editor/app.js

```diff
@@ -2159,7 +2159,8 @@ PTL.editor = {
 
     // FIXME: move this side-effect elsewhere
     if (results.length > 0 && results[0].source === sourceText) {
-      if (ReactEditor.stateValues[0] === '') {
+      const allowAutofill = this.settings.canSuggest || this.settings.canTranslate;
```

Comment:
Also I would rather calculate this on the python view and pass a single boolean var to JS code.